src/Craftify.Geometry.Domain/LineSegmentEvaluation.cs

```diff
@@ -11,4 +11,22 @@ public static Point3D GetPointAtParameter(this LineSegment lineSegment, double p
             start.Z + parameter * (end.Z - start.Z)
         );
     }
+
+    public static double GetLength(this LineSegment lineSegment)
```

Comment:
I think it would be nicer to have functions like that,

the first one that would calculate vector between 2 points

```csharp
public static Vector3D ByTwoPoints(Point3D start, Point3D end)
{
    return end.AsVector().Subtract(start.AsVector());
}
```

Then we could have functions like that:

```csharp
public static Vector3D GetNormalizedDirection(this LineSegment line) =>
    Vector.ByTwoPoints(line.Start, line.End).Normalize();

public static Vector3D GetDirection(this LineSegment line) =>
    Vector.ByTwoPoints(line.Start, line.End);
```

And then here is the way our GetLength would look like:

```csharp
public static double GetLength(this LineSegment line) =>
    line.GetDirection().Magnitude();
```

I already have that code written, would you like me to push that code?

I think it would be nicer to have functions like thi

The first one calculates the vector between two points:

```csharp
public static Vector3D ByTwoPoints(Point3D start, Point3D end)
{
    return end.AsVector().Subtract(start.AsVector());
}
```

Then, we could have functions like these:

```csharp

public static Vector3D GetNormalizedDirection(this LineSegment line) =>
    Vector.ByTwoPoints(line.Start, line.End).Normalize();

public static Vector3D GetDirection(this LineSegment line) =>
    Vector.ByTwoPoints(line.Start, line.End);

```

And here's how our `GetLength` function would look:

```csharp
public static double GetLength(this LineSegment line) =>
    line.GetDirection().Magnitude();

```

What do you think about it?

I already have this code written. Would you like me to push it?

Reply:
Looks more concise. We could push it through another PR before the current one.